solutions/usaco-1060.mdx

```diff
@@ -53,5 +53,83 @@ int main(){
 ```
 
 </CPPSection>
+	
+<JavaSection>
+	
+```java
+// Created by Sathvik Chundru with help from Justin Chen
+import java.util.*;
+import java.io.*;
+class Main {
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		int n = sc.nextInt();
+		int[] petals = new int[n];
+
+		for (int i = 0; i < n; i++) {
+
+			// storing each petal count in an array
+			// so we can keep track of which flower
+			// has how many petals based on the 
+			// index it is stored in
+
+			petals[i]=sc.nextInt();
+		}
+
+		int counter = 0;
+
+		// creating 2 nested arrays to brute force every 
+		// possible combination of flowers
+
+		for (int i = 0; i < n; i++) {
+			for (int j = i; j < n; j++) {
+
+				int flowerCount=j-i+1;
```

Comment:
whitespace between operators please

Reply:
fixed shit i forgot that one spot
coding guy pls don't hold against me in my team app am sorry I tried my best to get everything